third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -473,31 +536,39 @@ func resourceSqlDatabaseInstance() *schema.Resource {
 			"server_ca_cert": {
 				Type:     schema.TypeList,
 				Computed: true,
+				<% unless version == 'ga' -%>
```

Comment:
This change doesn't seem relevant to this PR- is it?

Reply:
I've tried to avoid diff in `state`

Reply:
Why was there a diff in state for a computed field?

Reply:
I've re-checked, it's not needed. Removed